src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -209,6 +210,13 @@ export enum MutationTableColumnType {
     DBSNP = 'dbSNP',
     GENE_PANEL = 'Gene panel',
     SIGNAL = 'SIGNAL',
+    NUM_MUTATED_GROUP_A = '(A) Group',
+    NUM_MUTATED_GROUP_B = '(B) Group',
```

Comment:
We may want these column names match the actual group names. Not sure if it is easy to implement though.

Reply:
Yea, it wouldn't be easy and I think keeping the general column name is good enough